plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/AbstractTestIcebergConnectorTest.java

```diff
@@ -30,10 +30,13 @@
 import io.trino.spi.statistics.ColumnStatistics;
 import io.trino.spi.statistics.DoubleRange;
 import io.trino.spi.statistics.TableStatistics;
-import io.trino.testing.AbstractTestIntegrationSmokeTest;
+import io.trino.testing.BaseConnectorTest;
```

Comment:
can you add a preparatory commit which introduces `AbstractTestIcebergConnectorTest.java` based on `TestIcebergConnectorTest` without yet merging `ATISmoke` into it